src/go/guestagent/pkg/tracker/apitracker_test.go

```diff
@@ -93,32 +98,41 @@ func TestAddOverride(t *testing.T) {
 	defer testSrv.Close()
 
 	apiTracker := tracker.NewAPITracker(context.Background(), &testForwarder{}, testSrv.URL, hostSwitchIP, true)
+
+	protoPort1, err := nat.NewPort(protocol, hostPort)
```

Comment:
For consistency with other usage, this should be `protoPort` (or `hostPort` should be `hostPort1`):

```suggestion
	protoPort, err := nat.NewPort(protocol, hostPort)
```